backend/src/middleware/login/loginMiddleware.ts

```diff
@@ -2,43 +2,45 @@
 /* eslint-disable @typescript-eslint/no-unsafe-member-access */
 /* eslint-disable @typescript-eslint/no-unsafe-call */
 /* eslint-disable @typescript-eslint/no-unsafe-assignment */
-import { sendMail } from '@middleware/helpers/email/sendMail'
 import {
-  signupTemplate,
-  resetPasswordTemplate,
-  wrongAccountTemplate,
-  emailVerificationTemplate,
-} from '@middleware/helpers/email/templateBuilder'
+  sendRegistrationMail,
+  sendEmailVerification,
+  sendResetPasswordMail,
+  sendWrongEmail,
+} from '@src/emails/sendEmail'
 
 const sendSignupMail = async (resolve, root, args, context, resolveInfo) => {
-  const { inviteCode } = args
+  const { inviteCode, locale } = args
   const response = await resolve(root, args, context, resolveInfo)
   const { email, nonce } = response
   if (nonce) {
     // emails that already exist do not have a nonce
     if (inviteCode) {
-      await sendMail(signupTemplate({ email, variables: { nonce, inviteCode } }))
+      await sendRegistrationMail({ email, nonce, locale, inviteCode })
     } else {
-      await sendMail(signupTemplate({ email, variables: { nonce } }))
+      await sendRegistrationMail({ email, nonce, locale })
     }
   }
   delete response.nonce
   return response
 }
 
 const sendPasswordResetMail = async (resolve, root, args, context, resolveInfo) => {
-  const { email } = args
+  const { email, locale } = args
   const { email: userFound, nonce, name } = await resolve(root, args, context, resolveInfo)
-  const template = userFound ? resetPasswordTemplate : wrongAccountTemplate
-  await sendMail(template({ email, variables: { nonce, name } }))
+  if (userFound) {
+    await sendResetPasswordMail({ email, nonce, name, locale })
+  } else {
+    await sendWrongEmail({ email, locale })
```

Comment:
Please remove this, this is an antifeature allowing to spam emails from our service

![image](https://github.com/user-attachments/assets/bdf4428c-2388-4013-ad5f-82bf69d61313)

```suggestion
    // this is an antifeature allowing unauthenticated users to spam any email with wrong-email notifications
    // await sendWrongEmail({ email, locale })
```

Reply:
Approved von @Tirokk

Reply:
Right